test/python/circuit/library/test_qaoa_ansatz.py

```diff
@@ -19,6 +19,7 @@
 from qiskit.circuit.library import HGate, RXGate, YGate, RYGate, RZGate
 from qiskit.circuit.library.n_local.qaoa_ansatz import QAOAAnsatz
 from qiskit.opflow import I, Y, Z, PauliSumOp
```

Comment:
Unused import (breaks lint)
```suggestion
from qiskit.opflow import I, Y, Z
```